internal/resource/config/oauth/authserversettings/scopes/commonscope/oauth_auth_server_settings_scopes_common_scope_resource.go

```diff
@@ -19,31 +19,31 @@ import (
 
 // Ensure the implementation satisfies the expected interfaces.
 var (
-	_ resource.Resource                = &oauthAuthServerSettingsScopesCommonScopesResource{}
-	_ resource.ResourceWithConfigure   = &oauthAuthServerSettingsScopesCommonScopesResource{}
-	_ resource.ResourceWithImportState = &oauthAuthServerSettingsScopesCommonScopesResource{}
+	_ resource.Resource                = &oauthAuthServerSettingsScopesCommonScopeResource{}
+	_ resource.ResourceWithConfigure   = &oauthAuthServerSettingsScopesCommonScopeResource{}
+	_ resource.ResourceWithImportState = &oauthAuthServerSettingsScopesCommonScopeResource{}
 )
 
-// OauthAuthServerSettingsScopesCommonScopesResource is a helper function to simplify the provider implementation.
-func OauthAuthServerSettingsScopesCommonScopesResource() resource.Resource {
-	return &oauthAuthServerSettingsScopesCommonScopesResource{}
+// OauthAuthServerSettingsScopesCommonScopeResource is a helper function to simplify the provider implementation.
+func OauthAuthServerSettingsScopesCommonScopeResource() resource.Resource {
+	return &oauthAuthServerSettingsScopesCommonScopeResource{}
 }
 
-// oauthAuthServerSettingsScopesCommonScopesResource is the resource implementation.
-type oauthAuthServerSettingsScopesCommonScopesResource struct {
+// oauthAuthServerSettingsScopesCommonScopeResource is the resource implementation.
+type oauthAuthServerSettingsScopesCommonScopeResource struct {
 	providerConfig internaltypes.ProviderConfiguration
 	apiClient      *client.APIClient
 }
 
-type oauthAuthServerSettingsScopesCommonScopesResourceModel struct {
+type oauthAuthServerSettingsScopesCommonScopeResourceModel struct {
 	Id          types.String `tfsdk:"id"`
 	Name        types.String `tfsdk:"name"`
 	Description types.String `tfsdk:"description"`
 	Dynamic     types.Bool   `tfsdk:"dynamic"`
 }
 
 // GetSchema defines the schema for the resource.
-func (r *oauthAuthServerSettingsScopesCommonScopesResource) Schema(ctx context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
+func (r *oauthAuthServerSettingsScopesCommonScopeResource) Schema(ctx context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
 	schema := schema.Schema{
 		Description: "Manages a OauthAuthServerSettingsScopesCommonScopes.",
```

Comment:
Looks like this needs to be something like "Manages an Oauth Auth Server Settings Scopes Common Scope"

Reply:
Updated both